File: project-server/Models/Days.cs
namespace project_server.Models
{
    public class Days
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public DateTime Day { get; set; }
        public int? MealTypeId { get; set; }
        public int ItemId { get; set; }
        public double Measurement { get; set; }
    }
}

File: project-frontend/src/models/Days.ts
export interface Days {
    id: number;
    userId: number;
    day: string;
    mealTypeId: number;
    itemId: number;
    measurement: number;
    name: string | null;
    proteins: number | null;
    fats: number | null;
    carbs: number | null;
    description: string | null;
    apiId: number | null;
    calories: number | null;
}
